config-base-next.js

```diff
@@ -20,5 +21,17 @@ module.exports = {
         'unicorn/no-null': 'warn',
       },
     },
+    {
+      files: ['**/test/**/*'],
+      rules: {
+        '@typescript-eslint/consistent-type-assertions': 'off',
+        '@typescript-eslint/explicit-function-return-type': 'off',
+        '@typescript-eslint/explicit-module-boundary-types': 'off',
+        '@typescript-eslint/no-empty-function': 'off',
+        '@typescript-eslint/require-await': 'off',
+        'unicorn/no-null': 'off',
+        'unicorn/no-useless-undefined': 'off',
```

Comment:
Any reason in particular this one is off?

Reply:
can't confirm since most of these are from Max's PR that I linked in the description, but I think these are `off` so the service that uses this package has the freedom to choose if they want it on or not so having it as `off` by default seems more intuitive